src/cogent3/core/alignment.py

```diff
@@ -940,7 +942,7 @@ def __add__(self, other):
         new.annotation_db = None
         return new
 
-    def add_seqs(self, other, before_name=None, after_name=None):
+    def add_seqs(self, other, before_name=None, after_name=None):  # will not port
```

Comment:
should we port? would have to think about what this means for SeqsData

Reply:
on SeqsData this operation should be supported, but it returns a new SeqsData instance. So that means adding `add_seqs()` to the ABC